composer.json

```diff
@@ -3,6 +3,16 @@
     "type": "sylius-plugin",
     "description": "Sylius Plugin for order creation in Admin panel",
     "license": "MIT",
+    "authors": [
+        {
+            "name": "Mateusz Zalewski",
+            "homepage": "http://mpzalewski.com.pl"
+        },
+        {
+            "name": "Grzegorz Sadowski",
+            "email": "sadowskigp@gmail.com"
```

Comment:
Shouldn't we use company emails in this plugin?

Reply:
I think it's up to the author, Grześ, for example, has this email in his Github account, so I understand he doesn't want to mess it up